### PR TITLE
Removed Python v3.8 support

### LINKED ISSUE #84: Remove Python v3.8 support
Python v3.8 is end-of-life (EOL) in 2024. Support for it should be removed.

### PR DESCRIPTION
**Pull request by gabb222**
Fixes #84 

Now supports Python versions >=3.9

### CODE REVIEW

**Comment by gabb222** [comment]
The check for Python v3.8 failed because support for it was removed. Should the check be removed/ignored or should support for v3.8 remain until official EOL?

**Comment by gabb222** [comment]
@volks73 @Apilny This PR is ready for review!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.12"]
+        python-version: ["3.9", "3.10", "3.12"]
```
Comment (volks73) [inline .github/workflows/ci.yml:18]: Let's add Python v3.11 to the list, too.

```yaml
python-version: ["3.9", "3.10", "3.11", "3.12"]
```

**Review by volks73 — APPROVED** [review]
Looks good, but let's add Python v3.11 to the test matrix for completeness.

Approving to avoid blocking and adding Python v3.11 can be a separate PR if needed.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (gabb222) [reply .github/workflows/ci.yml]: This change was made in the following commit

**Pull request merged**